rs/ic_os/config_types/src/lib.rs

```diff
@@ -38,6 +38,8 @@ pub static RESERVED_FIELD_NAMES: &[&str] = &[];
 
 pub type ConfigMap = HashMap<String, String>;
 
+pub const PROD_GUEST_VM_MEMORY: u32 = 490;
```

Comment:
As above, this should probably be closer to the VM crates, but I'm avoiding circular dependencies for now.